src/components/profile/ProfileTeamLocation.vue

```diff
@@ -258,7 +263,7 @@ export default {
 }
 .profile__team-location a,
 .profile__team-location strong {
-  display: block;
+  display: inline;
```

Comment:
This must be `inline-block` otherwise there's no space between location and office:  `LOCATION(OFFICE)` vs `LOCATION (OFFICE)`